### PR TITLE
Fix: Remnant: Lost in Space, Cog 24 Esquiline mention, and misc.

### PR DESCRIPTION
**Pull request by Zitchas**
**Bugfix:** This PR addresses multiple bugs and inconsistencies, most notably the Lost in Space continuing to be triggered even after the player has regained a method of escaping the Ember Waste; as well as a `<source>` that should be an `<origin>`

Thanks to Terin, Hurleveur, Arrow2thekn33, Salt Breaker, and a few others I'm probably missing for pointing these out.

## Fix Details
1. Make the Lost in Space mission check to ensure that the player hasn't managed to get a license or method of escape in the time since the timer was started.
2. Standardize the few remaining remnant events to start with "remnant:" so they sort into the right place.
3. replace the `<source>` with `<origin>`
4. Add a mention that the cargo needs to be picked up from Esquiline in Cognizance 24.

The events that are renamed here aren't tested outside of their adjacent mission. Just in case someone happens to be *inside* these very mission sequences when they get this update, there is a compatibility built into them. Likewise, there is a compatibility patch so that the few people who ever get this mission won't risk getting it twice. (never mind that merely getting the basic license, which they should be getting if they're spending this much time around the Remnant, will block any such potential double anyway.)

### CODE REVIEW

File: data/remnant/remnant 1 introduction.txt [hunk 1]
```diff
@@ -150,7 +150,7 @@ mission "Remnant Hails Change"
 
 
 
-mission "lost in ember waste"
+mission "Remnant: lost in ember waste"
```
Comment (tehhowch) [inline data/remnant/remnant 1 introduction.txt:153]: Don't change mission identifiers after they've been released unless absolutely necessary.

Everyone who already experienced the mission will be eligible for it again.

File: data/remnant/remnant 1 introduction.txt [hunk 2]
```diff
@@ -193,7 +197,7 @@ mission "Remnant: Lost in Ember Waste"
 				goto keystone
 			
 			label know
-			`	"Get scammed out of your Key Stone?" she asks with another chuckle. "Or maybe your jump drive, and now you're stuck here? No problem."`
+			`	"Lost your Key Stone?" she asks with another chuckle. "Or maybe your jump drive, and now you're stuck here? No problem."`
```
Comment (tehhowch) [inline data/remnant/remnant 1 introduction.txt:200]: Why change this?

**Review by tehhowch — CHANGES_REQUESTED** [review]
Revert the breaking changes to mission and event identifiers. These names are visible to plugins and have been released for ages. Changing them and maintaining compatibility requires continuing to both set and trigger off the previous identifiers.

File: data/remnant/remnant 1 introduction.txt (hunk 1 above)
Comment (Zitchas) [reply data/remnant/remnant 1 introduction.txt:153]: As it stands, this is one of the least-seen missions in the history of the game. Someone accidentally getting it a second time is only likely to happen if someone *tries* to get it. But sure, I can fix this.

File: data/remnant/remnant events.txt [hunk 3]
```diff
@@ -21,12 +21,12 @@ event "remnant change hail"
...
 
 
-event "lost in ember waste"
+event "remnant: lost in ember waste"
 
 
 
-event "remnant defense timer"
-	clear "Remnant defense delay"
+event "remnant: defense timer"
```
[the remark this answers is not in the document]
Comment (Zitchas) [reply data/remnant/remnant events.txt:28]: This falls into the category of "Polishing the content to make it less "thrown together beta" and more suitable for our eventual "professional looking game" 1.0 release.

[the remark this answers is not in the document]
Comment (Zitchas) [reply data/remnant/remnant events.txt:28]: Tehhowch, you regularly push people to "reduce the tech debt" and "improve the codebase to remove problems." This is exactly the same sort of thing, except on the content side. 

As far as I am aware, there is not a single plug-in or piece of content anywhere other than this very mission that refers to this. With this changed so it actually is collected where it *should* be, chances are I will actually at least refer to it in a conversational branch somewhere. On the other hand, with it being an oddball thing that doesn't match properly, I can pretty much guarantee I'll never use it. Out of sight, out of mind.

File: data/remnant/remnant 1 introduction.txt (hunk 1 above)
Comment (Zitchas) [reply data/remnant/remnant 1 introduction.txt:153]: Has now been fixed. People cannot get this mission a second time.

File: data/remnant/remnant 1 introduction.txt [hunk 4]
```diff
@@ -127,42 +127,59 @@ mission "First Contact: Remnant"
...
+mission "Remnant: lost in ember waste compatibility"
+	landing
+	invisible
+	source
+		government "Remnant"
+	to offer
+		has "lost in ember waste: declined"
+	on offer
+		set "Remnant: Lost in Ember Waste: offered"
+		set "Remnant: Lost in Ember Waste: declined"
```
Comment (Amazinite) [inline data/remnant/remnant 1 introduction.txt:181]: The mission name still has lowercase "lost in ember waste", yet this compatibility mission uses capitalized words.

Comment (Zitchas) [reply data/remnant/remnant 1 introduction.txt:182]: Fixed. Thanks for pointing that out.

**Review by Amazinite — CHANGES_REQUESTED** [review]
The actual bug fixes are distinct from the renaming of missions, events, and conditions, so I'd rather this PR not deal with both at once. As for the renaming though, I don't see a need for it; we should really only be renaming things if it's necessary to do so, not renaming them for the sake of renaming them. I'm reminded of #5208, where there was a complaint that the conditions for checking if a player has met a faction weren't the same across all factions; it isn't necessary that the conditions have a standardized naming scheme, only that conditions that represent what you want them to are available. (That isn't the same thing as saying we should have no naming scheme, but given the overhead of retroactively enforcing a naming scheme, we shouldn't go about doing that.)

If we want to do any sort of "clean up/standardize missions/conditions" pass for a 1.0 release, then let's actually do it during said 1.0 release instead of making such changes now and having to deal with a mess of unnecessary compatibility missions until we get there.

**Comment by Zitchas** [comment]
Based on the assumption that the community fried and people continue to come and go and may not necessarily be around to update their stuff whenever 1.0 happens, I think a better approach is to make the change now so people can build stuff properly, and remove the compatibility stuff when we get to 1.0.

It is better to build it right from the earliest point possible than to continue to build something wrong on the assumption it will be fixed at some later date. Especially since that later date could well be a decade from now. Or might never happen at all.

Worth noting that up until now I am the one doing the bay majority of Remnant content, and this makes my work easier. Likewise, my additions have generated increased interest and others are starting to think about contributing stuff too. I would prefer to make the Remnant stuff as correct as possible so that people can build on it without needing to add to the burden of stuff needing to be adjusted in the mythical 1.0 release.

**Comment by Amazinite** [comment]
That doesn't change the fact that the actual bug fixes are distinct from the renamings. Split them up. 

> I think a better approach is to make the change now so people can build stuff properly...  I would prefer to make the Remnant stuff as correct as possible so that people can build on it without needing to add to the burden of stuff needing to be adjusted in the mythical 1.0 release.

Realize that we're talking about a minute number of names compared to the entirety of Remnant content. It's not as if this small handful of mission and condition names that other people wrote that already match naming standards from other areas of the game will cripple future content creation for the Remnant just because you've chosen a separate standard for what now makes up the vast majority of Remnant content. It's also not as if we won't be able to maintain a standard for Remnant content going forward as you have already done when new content is PRed just because these outliers exist.

And as I've already mentioned, the exact names used don't even matter, it's all about what they represent; that's why I brought up #5208. We should look at it in terms of "Which conditions do I need to use to get the job done?" not "How do I make it so that all the conditions I use look the same?"

I'll also note that in the comment for your "Tech Debt Clean-Up" commit you mention readability of the save file, which a week before that commit I already described to you isn't a goal we should worry ourselves with. But insofar as we're discussing readability, we should concern ourselves more with the readability of the data files, not the readability of the save file, and I'd say having a bunch of compatibility missions and duplicate conditions with minor changes to them just to fit a standard that wasn't applied previously unnecessarily detracts from the readability of the data files.

**Comment by Zitchas** [comment]
**Save-readability**
This is a community-run and community-driven project. We actively encourage people to read the data-files, and for most people, that journey starts with reading their save-file. As such, it is in our own best interests to make the save-file as easy to understand as possible so that people doing so can actually build up some confidence in what they are reading. Part of that includes making it, to the best of our own ability, so that everything pertaining to a particular mission set is in the same area.

Ease of save reading isn't some unsupported miscellaneous thing: It is a fundamental means of growing our community. Every simple bit of hardship adds to the difficulty level, and discourages someone. The easier we make it, the more people we retain and the more eyes on the code.

**Data file readability**
In terms of complexity, a couple compatibility missions adds virtually zero complexity to the game. They are *extremely* simple. They are among the most basic mission structures in the game, and I doubt they confuse anyone. In fact, they *simplify* the game, as people only have to look at a single spot in their save to see all Remnant-related variables, which makes it easier for people to write new content. And we shouldn't pretend that save files aren't part of the game. They are a huge part of it. Everyone writing missions has to go look at save files. Repeatedly. And everyone testing has to. And everyone troubleshooting has to. We can't just ignore them as some inconsequential detritus of no consequence to the game.

If "maintaining the readability of the data files" is truly an overriding concern, than our our top priority is, unquestionably, to re-write the entirety of the Remnant-Deep interaction missions in order to have each mission's conversation listed normally inside it. Now *that* is a readability problem. I believe you yourself said that you need to write out an entire table of values just to keep track of what's going on there. And we expect people who have never looked at the data before to try to figure out exactly what they need to do in order to unlock the Bactrian? I consider those missions to be a wonder of programming and mission writing, demonstrating the potential of our engine. But readable? Not in the slightest.

**Coding facilitation**
1. If all the Remnant code is formatted identically, then it makes it much easier to write, as one doesn't have to remember a variety of formats. 
2. Likewise, consistent formatting makes it easier to check to ensure that there are no mistakes during the review process. 
3. It makes it easier for new writers to engage with the code as all the relevant variables will be stored in the same place in their save, so it will be easier for them to figure out what they should use to base their new content on.

**Summary**
Everything in here are fixes. Some are definitely more urgently requested than others, but they are all necessary in my view to bring the small piece of the game that I am responsible for up to standard and improve everything for those who follow. I am not interested in having chunks of it shunted off into separate PRs that will immediately be closed as "won't fix" because this is something that you don't see as a problem because it doesn't affect you. Or just as bad, labelled with a "1.0.0" milestone and abandoned to the ever increasing backlog.

The fix is here, ready to go. It adds a minuscule amount of very clear, unambiguous code to a file that is part of a section of the game you aren't particularly involved in. It doesn't affect you. It *will* improve the lives of all of us who troubleshoot people's saves or write Remnant content.

**Comment by Amazinite** [comment]
> If "maintaining the readability of the data files" is truly an overriding concern, than our our top priority is, unquestionably, to re-write the entirety of the Remnant-Deep interaction missions in order to have each mission's conversation listed normally inside it.

There's a distinction between readability and complexity to build. Things can be complex to build but once finished simple to read, and so long as you can read how branches work then there isn't anything unreadable about that mission.

**Pull request merged**